src/ansi-c/ansi_c_entry_point.cpp

```diff
@@ -505,8 +481,11 @@ bool ansi_c_entry_point(
   else
   {
     // produce nondet arguments
-    call_main.arguments()=
-      build_function_environment(parameters, init_code, symbol_table);
+    call_main.arguments()=build_function_environment(
```

Comment:
I have added an argument (message_handler, to make it more like the java code), so it no longer fits into one line if I wrap after the `=`.